### PR TITLE
GitHub & Trello usage procedures

### PR DESCRIPTION
**Pull request by slamora**
Added documentation of the procedures of GitHub and Trello.

### CODE REVIEW

File: github-usage.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# GitHub - usage
+
+1. Generally, on branch per issue (read [branch-naming-convention](git-branch-naming-convention.md)).
```
Comment (dunetna) [inline github-usage.md]: Generally, **one** branch per issue

File: github-usage.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    Lifecycle of the branch: create branch --> create PR --> review --> (apply changes) --> merge --> delete branch
+    - PR title: descrive in a few words what fix/implements this pull request
```
Comment (dunetna) [inline github-usage.md]: PR title: **describe** in a few words what **this pull request fixes/implements**

File: trello-usage.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Trello usage
+
+Usually one card per task (if there are related task you can link the cards as attachments); or even create an `Epic` card to group several related cards.
```
Comment (dunetna) [inline trello-usage.md]: if there are related **tasks**

File: trello-usage.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+1. Before starting a task (card), read carefully the description and comments. **Ask if you have questions** before start writing code.
+2. Keep updated the card column according to its state (backlog, to-do, doing, testing...).
+3. Document the changes of the task on the card (leave comments, update checklists...).
+4. Add links to related PR/issues of GitHub; if you are working on a GitHub issue/PR that doesn't have a card on trello just create a new card.
+
+### Description of the trello columns meaning
+- **Backlog**: detected tasks which has not been scheduled yet.
+- **Design**: tasks which specs are being defined but has not been scheduled yet.
```
Comment (dunetna) [inline trello-usage.md]: tasks **whose** specs are being defined but **have** not been scheduled yet.

File: trello-usage.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+- **To-do**: tasks with completed specs which has been scheduled for the current sprint.
```
Comment (dunetna) [inline trello-usage.md]: tasks with completed specs **that have been** scheduled

File: trello-usage.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+- **Doing**: tasks which the team is working on it.
```
Comment (dunetna) [inline trello-usage.md]: **tasks the team is working on.**

**Review by dunetna — APPROVED** [review]
I have only made a few language suggestions.
Thanks for this documentation!

**Review by cayop — COMMENTED** [review]
In github-usage.md:
I think is good idea add check the test in 2 points.
1.- next the 3th point normaly is common have some test for check this new codes with an example than works and other of extremes of casuistry than it doesn't work.
2.- specifically, when there are a backend bug, there are always a test then reproduce this bug with the solution working. (It isn't necessary for hotfix)

**Pull request merged**